evalml/tests/data_checks_tests/test_target_leakage_data_check.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+import pytest
+
+from evalml.data_checks.data_check_message import DataCheckWarning
+from evalml.data_checks.target_leakage_data_check import TargetLeakageDataCheck
+
+
+def test_label_leakage_data_check_init():
```

Comment:
Not pressing but maybe change name to `test_target_leakage_data_check_init` and etc for the test names?